src/components/game-setup/GameStepper.js

```diff
@@ -0,0 +1,137 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Stepper from '@mui/material/Stepper';
+import Step from '@mui/material/Step';
+import StepButton from '@mui/material/StepButton';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
```

Comment:
```suggestion
import { Box, Stepper, Step, StepButton, Button, Typography } from '@mui/material';
```

You can actually import everything with one line!